mutation_partition_view.cc

```diff
@@ -276,6 +276,122 @@ future<> mutation_partition_view::do_accept_gently(const column_mapping& cm, Vis
     }
 }
 
+template <bool is_preemptible>
+mutation_partition_view::accept_ordered_result mutation_partition_view::do_accept_ordered(const schema& s, mutation_partition_view_virtual_visitor& visitor, accept_ordered_cookie cookie) const {
+    auto in = _in;
+    auto mpv = ser::deserialize(in, boost::type<ser::mutation_partition_view>());
+    const column_mapping& cm = s.get_column_mapping();
+
+    if (!cookie.accepted_partition_tombstone) {
+        mpvlog.trace("do_accept_ordered: accept partition tombstone: {}", tombstone(mpv.tomb()));
```

Comment:
Do we really need logging here? I don't think we have it in any other producers.

Reply:
I needed it for development.  I presumed that the runtime penalty is negligible if the log level isn't high enough, so nothing gets computed.